src/platform/user/authorization/components/IdentityNotVerified.jsx

```diff
@@ -14,29 +14,52 @@ export const HowToVerifyLink = () => (
   </p>
 );
 
+const VerifyIdentityInfo = () => (
+  <va-additional-info
+    trigger="If you have trouble verifying your identity"
+    uswds
+  >
+    <a
+      href="https://www.va.gov/resources/verifying-your-identity-on-vagov/"
+      clasName="vads-u-padding-bottom--2"
+    >
+      Get answers to common questions about verifying your identity
+    </a>
+    <div className="vads-u-margin-top--2p5">
+      <p>
+        Or, if you have a Premium My HealtheVet account with a My HealtheVet
+        user ID and password, you can sign out and then sign back in with that
+        account to access My HealtheVet.
+      </p>
+    </div>
+  </va-additional-info>
+);
+
 const IdentityNotVerified = ({
   headline = 'Verify your identity to access your complete profile',
   showHelpContent = true,
+  showVerifyIdenityHelpInfo = false,
 }) => {
   return (
     <>
-      <va-alert status="continue" class="vads-u-margin-top--3">
+      <va-alert
+        status="continue"
+        class="vads-u-margin-top--3 vads-u-margin-bottom--3"
+      >
         <h2 slot="headline" data-testid="verify-identity-alert-headline">
           {headline}
         </h2>
-
         <div className="vads-u-margin-bottom--1">
           <p>
-            We need to make sure you’re you&#8212;and not someone pretending to
-            be you&#8212;before we can give you access to your personal and
-            health-related information. This helps to keep your information
-            safe. It helps to prevent fraud and identity theft.
+            Our records show that you haven’t verified your identity for your
+            ID.me account. We need you to verify your identity for this account
+            to help us keep your information safe and prevent fraud and identity
+            theft.
           </p>
-
-          <p className="vads-u-font-weight--bold">
-            This one-time process takes about 5-10 minutes.
+          <p>
+            ID.me will ask you for certain personal information and
```

Comment:
Okay so basically anywhere it says `ID.me` needs to be dynamic which you can just pass down as props from the `LandingPage` component since it already has `signInService`.

Here are the conditions for `signInService`
- If it is `mhv` or `dslogon`, the content needs to be updated to include `Login.gov or ID.me`
- If it is `idme`, the content needs to be updated to say `ID.me`
- If it is `logingov`, the content needs to be updated to say `Login.gov`

I'm happy to sync as well.

Reply:
Updates are in